### ISSUE TITLE
Add `tauri-cli` for various architectures to standard release

### ISSUE THREAD

**Issue by littlebenlittle** [tauri-apps/tauri#2421]
**Describe the solution you'd like**

It would be useful for the standard release cycle to include signed binaries of `tauri-cli` for various architectures.

Distributing binaries prevents redundant compilation of the cli tool across CI pipelines and projects.

**Describe alternatives you've considered**

The alternative is each project building the cli during CI, which is not a great use of computing resources.

**Comment by lucasfernog** [comment]
Can you list the architectures you need?

**Comment by littlebenlittle** [comment]
`aarch64-unknown-linux-musl` is my current target, but I can imagine needing others in the future for android and IOS.

**Comment by lucasfernog** [comment]
Yeah we'll take care of mobile for v2 probably.

**Comment by amrbashir** [comment]
this should be fixed now with `@tauri-apps/cli@1.0.0-rc.0` right?

**Comment by lucasfernog** [comment]
Yeah, here's the current list:

```
x86_64-apple-darwin
aarch64-apple-darwin
x86_64-pc-windows-msvc
x86_64-unknown-linux-gnu
x86_64-unknown-linux-musl
aarch64-unknown-linux-gnu
armv7-unknown-linux-gnueabihf
aarch64-unknown-linux-musl
```
